packages/react/src/components/builder-component.component.tsx

```diff
@@ -126,37 +126,149 @@ const Throttle = decorator(throttle);
 const fetchCache: { [key: string]: any } = {};
 
 export interface BuilderComponentProps {
+  /**
+   * @package
+   * @deprecated use {@link name} instead.
+   */
   modelName?: string;
+  /**
+   * Name of the model this is rendering content for. Default's 
+   */
   model?: string;
+  /**
+   * @package
+   * @deprecated use {@link name} instead.
+   */
   name?: string;
+  /**
+   * @see {@link https://github.com/BuilderIO/builder/tree/master/packages/react#passing-data-and-functions-down}
+   */
```

Comment:
Could we add a tiny bit more prose to at least help decide if the reader should follow the link or not?

Reply:
Can do